__tests__/app.test.js

```diff
@@ -127,6 +127,49 @@ describe('/api/articles', () => {
             body.articles.forEach((article) => {
               expect(typeof article.comment_count).toBe('number')
             })
+        })
+    })
+})
+
+describe('/api/articles/:article_id/comments', () => {
+    test('GET: 200 should return an array of all comments for the supplied article id. Each comment should have the following properties: comment_id, votes, created_at,author, body & article_id', () => {
+        return request(app)
+          .get('/api/articles/1/comments')
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.comments).toHaveLength(11)
+            body.comments.forEach((comment) => {
+              expect(typeof comment.comment_id).toBe('number')
+              expect(typeof comment.votes).toBe('number')
+              expect(typeof comment.created_at).toBe('string')
+              expect(typeof comment.author).toBe('string')
+              expect(typeof comment.body).toBe('string')
+              expect(typeof comment.article_id).toBe('number')
+            })
+        })
+    })
+    test("GET: 200 returned comments should be sorted by most recent first", () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+          expect(body.comments).toBeSortedBy('created_at',{descending: true})
+        })
+      })
```

Comment:
One test missing here:
It's possible for an article to have no comments associated with it (look in your data for an example of this)
In this case, we would expect (200) and we would expect an empty array to be returned